providers/shared/references/Processor/id.ftl

```diff
@@ -100,6 +100,16 @@
             ] +
             nodeCountChildConfiguration
         },
+        {
+            "Names" : [ "ddb", "DDB" ],
+            "Children" : [
+                {
+                    "Names" : "Processor",
+                    "Types" : STRING_TYPE
+                }
+            ] +
+            nodeCountChildConfiguration
+        },
```

Comment:
Don't need to do casing here as we already do a to lower on it, and we don't really need two ways to set it. 

This should also be updated to align with the component name if that is changed.

Reply:
I just replicated the pattern from other components in case it mattered. Done